src/transformStateWithClones.js

```diff
@@ -8,6 +8,37 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  let newState = { ...state };
+  const array = [];
```

Comment:
```suggestion
  const statesHistory = [];
```
array/arr/obj/object/el/elem/res/result and so on are bad names. The name should describe variable